include/mbedtls/arc4.h

```diff
@@ -19,10 +19,22 @@
  *  limitations under the License.
  *
  *  This file is part of mbed TLS (https://tls.mbed.org)
+ *
+ * \deprecated   ARC4 is considered a weak cipher and its use
+ *               constitutes a security risk. It is recommended
+ *               alternative ciphers should be considered instead.
+ *
  */
 #ifndef MBEDTLS_ARC4_H
 #define MBEDTLS_ARC4_H
 
+#if !defined(MBEDTLS_DEPRECATED_REMOVED)
+
+#if defined(MBEDTLS_DEPRECATED_WARNING)
+#warning ARC4 is deprecated and its use constitutes a security risk.
+#warning It is recommended alternative ciphers should be considered instead.
```

Comment:
Having multiple `#warning` directives that shoukd read as a single paragraph is not very nice. GCC, at least, shows each directive separately. Please use a single directive (single line, or multiline string literal, or backslash continuations, whatever).

Reply:
Done.